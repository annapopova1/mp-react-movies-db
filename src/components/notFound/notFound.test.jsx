import React from 'react';
import NotFound from './notFound';

describe('<NotFound/>', () => {
  test('should render NotFound component', () => {
    const notFound = shallow(<NotFound />);
    expect(notFound).toMatchSnapshot();
  });
});
