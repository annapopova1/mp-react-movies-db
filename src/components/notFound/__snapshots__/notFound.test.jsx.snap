// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound/> should render NotFound component 1`] = `
<div
  className="alert alert-danger"
>
  The page is not found. Make sure you are using a correct URL.
</div>
`;
